libs/platform/src/lib/components/list/standard-list-item/standard-list-item.component.html

```diff
@@ -37,13 +37,13 @@
     <fd-avatar class="fd-list__icon" size="s" [image]="avatarSrc" *ngIf="avatarSrc !== null && avatarSrc !== undefined"
         [attr.alt]="avatarTitle" [label]="avatarTitle" role="img">
     </fd-avatar>
-    <span *ngIf="title" fd-list-title [attr.aria-label]="unRead? 'unread ' + title: title" [attr.nowrap]="titleWrap"
+    <span *ngIf="title" fd-list-title i18n-aria-label="@@standardList.Item.unReadTitle" [attr.aria-label]="unRead? 'unread ' + title: title" [attr.nowrap]="titleWrap"
```

Comment:
As far as I can see  `unRead` is optional and based on it we constract the aria label. In the same time i18n version is hardcoded...
I don't see the posibility to constract i18n attribute using variable... As option we could use aria-labelledby and refer to the hidden elements with the correct content based on `unRead`.
The same for other places with the similar issue

Reply:
as discussed, added where ever it is required. here it is removed as text will be passed from application team.